pkgs/development/python-modules/yahooweather/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "yahooweather";
+  version = "0.10";
+
```

Comment:
This probably needs to be disabled for Python 2 as well: https://github.com/pvizeli/yahooweather/blob/master/setup.py#L27